compiler/damlc/tests/daml-test-files/Generics.daml

```diff
@@ -1,6 +1,8 @@
 -- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
+-- @INFO Unused LANGUAGE pragma
```

Comment:
Which one is unused? Can we just remove the unnecessary pragma here please?

Reply:
Done